working_notebook-TO_COMPLETE_Anna.ipynb

```diff
@@ -815,19 +815,25 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #31.            if patient_counter &gt; 0:
</pre><p>If you want to <code>raise</code>  (or <code>throw</code>  in Java terms) an Exception, there is the <code>raise</code>  statement in Python like</p><p> </p><pre class="ql-syntax" spellcheck="false">[...]
if patients_counter &gt; 0:
    raise ValueError("Number of patients do not match")
</pre><p> </p><p>However, those you're trying to verify are <strong>POST</strong> conditions, that is conditions should be verified <em>after</em> data have been read from the file. </p><p>In fact, you're capping the number of patients to <code>60</code>  by default. What If I pass you a different file with <code>125</code>  patients. The error condition won't hold. </p><p> </p><p>To verify POST conditions, testing is the way.. and so the second part of the exercise :)</p><p> </p><p>All in all, well done with the changes to the function!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WebValley2021ReImagined/programming-for-data-science/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='68'/>